### PR TITLE
Translated some pages into German

### PR DESCRIPTION
**Pull request by john2ksonn**
The pages are converter.md, flasher.md, powered.md, support.md, toolbox.md

### CODE REVIEW

**Comment by noroadsleft** [comment]
@kuchosauronad0 Can you do your magic here please?

File: _i18n/de/powered.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Die QMK Firmware Schutzmarke verwenden"
+layout: qmk
+permalink: /de/powered/
+---
+# Die QMK Schutzmarke
+
+"QMK Firmware" und das Logo sind von Jack Humbert registrierte Schutzmarken, um Tastaturen-Hersteller zu ermutigen zur Community beizutragen und der Community Unterstützung für Tastaturen mit QMK zu bieten.
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
"QMK Firmware" und das Logo sind von Jack Humbert registrierte Schutzmarken, um Tastaturen-HerstellerInnen zu ermutigen zur Community beizutragen und der Community Unterstützung für Tastaturen mit QMK zu bieten.
```

File: _i18n/de/powered.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Jeder ist berechtigt und ermutigt das Logo, die Begriffe "QMK (Firmware)", "Powered by QMK (Firmware)" (oder ähnlich Formulierungen mit "running", "compatible with", etc.) verwenden. Dem wird vorausgesetzt, dass:
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
Alle sind berechtigt und ermutigt das Logo, die Begriffe "QMK (Firmware)", "Powered by QMK (Firmware)" (oder ähnlich Formulierungen mit "running", "compatible with", etc.) verwenden. Dem wird vorausgesetzt, dass: 
```

File: _i18n/de/powered.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+layout: qmk
+permalink: /de/powered/
+---
+# Die QMK Schutzmarke
+
+"QMK Firmware" und das Logo sind von Jack Humbert registrierte Schutzmarken, um Tastaturen-Hersteller zu ermutigen zur Community beizutragen und der Community Unterstützung für Tastaturen mit QMK zu bieten.
+
+Jeder ist berechtigt und ermutigt das Logo, die Begriffe "QMK (Firmware)", "Powered by QMK (Firmware)" (oder ähnlich Formulierungen mit "running", "compatible with", etc.) verwenden. Dem wird vorausgesetzt, dass: 
+
+1. Nicht impliziert wird, dass QMK oder Beitragende die Tastatur befürworten / herstellen
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
1. Nicht impliziert wird, dass QMK oder Mitwirkende die Tastatur befürworten oder herstellen
```

File: _i18n/de/powered.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+2. [Der Hersteller seine Konfigurationen der Tastatur zu qmk:master zeitlich nach Versand der Tastaturen beitragen](https://github.com/qmk/qmk_firmware/pulls/)
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
2. [Der/Die HerstellerIn die Konfiguration der Tastatur zeitnah zum Release der Tastatur zu qmk:master beitragen](https://github.com/qmk/qmk_firmware/pulls/)
```

File: _i18n/de/powered.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+3. Auf der Produkt-Webseite auf die [QMK-Webseite](https://qmk.fm) oder das [GitHub-Repository](https://github.com/qmk/qmk_firmware) verlinkt wird
+
+Bilder mit dem Logo und bevorzugten Formulierungen zur Verwendung auf PCBs, Ausdrucken, Webseiten, Stickers, Fan-Artikeln, etc. sind unten zu finden.
+
+## Optionale Schritte
+
+1. Einen Etikett / Tag (`<keyboard>-<version>`z.B. `planck-4.2`) von dem geliefertem Commit mit der Tastatur-Firmware in [qmk/qmk_firmware](https://github.com/qmk/qmk_firmware) erfragen.
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
1. Beantragen Sie ein Etikett / Tag (`<keyboard>-<version>`z.B. `planck-4.2`) für das Commit mit dem die Tastatur in [qmk/qmk_firmware](https://github.com/qmk/qmk_firmware) ausgeliefert wird.
Üblicherweise wird für das Tag eine kompilierte .hex/.bin Datei auf der Release-Seite hinzugefügt.
```

File: _i18n/de/powered.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+2. Sich über [GPL Einhaltung](https://fsfe.org/activities/ftf/useful-tips-for-vendors.de.html) bewusst sein - Dabei hilft es ein Etikett / Tag zu haben.
+
+Wenn Ihr Projekt QMK als die standardmäßige und offizielle Firmware verwendet, würden wir es gerne zu der Liste auf unserer Webseite hinzufügen. Also [geben Sie bescheid, indem ein Pull Request auf qmk/qmk.fm](https://github.com/qmk/qmk.fm/pulls/) öffnen, sollte es noch nicht dort sein!
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
Wenn Ihr Projekt QMK als die standardmäßige und offizielle Firmware verwendet, würden wir es gerne zu der Liste auf unserer Webseite hinzufügen. Also [geben Sie uns Bescheid, indem Sie ein Pull Request auf qmk/qmk.fm](https://github.com/qmk/qmk.fm/pulls/) eröffnen, sollte es noch nicht dort sein!
```

File: _i18n/de/powered.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+</style>
+
+<table>
+    <tr>
+        <td><a href="/assets/images/badge-dark.svg"><img src="/assets/images/badge-dark.svg" alt="QMK Badge Dark" /></a></td>
+        <td><a href="/assets/images/badge-light.svg"><img src="/assets/images/badge-light.svg" alt="QMK Badge Light" /></a></td>
+    </tr>
+</table>
+
+A smaller version of each (pngs: [dark](/assets/images/badge-small-dark.png), [light](/assets/images/badge-small-light.png)):
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
 
```

File: _i18n/de/powered.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+Eine jeweils kleinere Version (PNGs: [dunkel](/assets/images/badge-small-dark.png), [hell](/assets/images/badge-small-light.png)):
+
+<table>
+    <tr>
+        <td><a href="/assets/images/badge-small-dark.svg"><img src="/assets/images/badge-small-dark.svg" alt="QMK Badge Small Dark" /></a></td>
+        <td><a href="/assets/images/badge-small-light.svg"><img src="/assets/images/badge-small-light.svg" alt="QMK Badge Small Light" /></a></td>
+    </tr>
+</table>
+
+And badges for compatible or community-supported keyboards (pngs: [dark](/assets/images/badge-community-dark.png), [light](/assets/images/badge-community-light.png)):
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
 
```

File: _i18n/de/powered.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+Und Badges für kompatible oder *von-der-Community-unterstützte*n Tastaturen (PNGs:  [dunkel](/assets/images/badge-community-dark.png), [hell](/assets/images/badge-community-light.png)):
+
+<table>
+    <tr>
+        <td><a href="/assets/images/badge-community-dark.svg"><img src="/assets/images/badge-community-dark.svg" alt="QMK Community Badge Dark" /></a></td>
+        <td><a href="/assets/images/badge-community-light.svg"><img src="/assets/images/badge-community-light.svg" alt="QMK Community Badge Light" /></a></td>
+    </tr>
+</table>
+
+**Bitte verwenden Sie keine bearbeiteten Versionen dieser Bilder** - Wenn Sie etwas anderes verwenden möchten, [lassen Sie es uns wissen](https://github.com/qmk/qmk.fm/issues), und wir werden unser bestes tun Ihren Wünschen nachzukommen.
```
Comment (kuchosauronad0) [inline _i18n/de/powered.md]:
```suggestion
**Bitte verwenden Sie keine bearbeiteten Versionen dieser Bilder** - Wenn Sie etwas anderes verwenden möchten, [lassen Sie es uns wissen](https://github.com/qmk/qmk.fm/issues), und wir werden unser Bestes tun Ihren Wünschen nachzukommen.
```

File: _i18n/de/support.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Unterstütze QMK"
```
Comment (kuchosauronad0) [inline _i18n/de/support.md]:
```suggestion
title: "Unterstützen Sie QMK"
```

File: _i18n/de/support.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+layout: qmk
+permalink: /de/support/
+---
+
+# Unterstütze QMK
```
Comment (kuchosauronad0) [inline _i18n/de/support.md]:
```suggestion
# QMK unterstützen
```

File: _i18n/de/support.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+* [QMK Fan-Artikel auf olkb.com](https://olkb.com/parts)
+* [Teespring Online-Shop](https://teespring.com/de/stores/qmk)
+* [Spenden Sie via Donorbox](https://donorbox.org/qmk)
+* [Wirke mit!](https://github.com/qmk/qmk_firmware/issues)
+
+## Was machen wir mit Spenden?
```
Comment (kuchosauronad0) [inline _i18n/de/support.md]:
```suggestion
## Wofür wir die Spenden verwenden
```

File: _i18n/de/support.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+
+* Helfen, die Server- und Infrastruktur-Kosten zu decken (letztendlich für den QMK Konfigurator)
```
Comment (kuchosauronad0) [inline _i18n/de/support.md]:
```suggestion
* Spenden helfen uns, die Server- und Infrastruktur-Kosten zu decken (letztendlich für den QMK Konfigurator)
```

File: _i18n/de/support.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+* Helfen, die Entwicklungs-Kosten für QMK-branded Produkte, wie den [Proton C](https://qmk.fm/proton-c)
```
Comment (kuchosauronad0) [inline _i18n/de/support.md]:
```suggestion
* Die Entwicklungs-Kosten für Prototypen anderer Produkte der QMK Marke zu decken, wie z.B. den [Proton C](https://qmk.fm/proton-c)
```

File: _i18n/de/support.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+* Fan-Artikel zu Treffen als Werbegeschenke / Preise senden. Schreiben Sie uns eine E-Mail, falls Sie interessiert sind! hello at-Symbol qmk.fm
+* [QMK Info Pakete](https://i.imgur.com/EoXgApN.png) für Tastaturen-Hersteller kaufen. (Optional $0.10 Spende, um die Kosten zu decken) - Schreiben Sie eine E-Mail, falls Sie Interesse haben!
```
Comment (kuchosauronad0) [inline _i18n/de/support.md]:
```suggestion
* [QMK Info Pakete](https://i.imgur.com/EoXgApN.png) für Tastatur-HerstellerInnen zu kaufen - Schreiben Sie eine E-Mail, falls Sie Interesse haben (Optional $0.10 Spende, um die Kosten zu decken)!
```

File: _i18n/de/toolbox.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+
+ - USBTiny (AVR Pocket)
+ - AVRISP (Arduino ISP)
+ - USBasp (AVR ISP)
+
+Sollte Interesse für andere bestehen, können sie hinzugefügt werden, wenn ihre Befehle bekannt sind. 
+
+## HID Konsole
+
+Die Toolbox horcht auch auf HID Nachrichten auf usage page `0xFF31` und usage `0X0074`. Dies ist kompatibel mit PJRC's [`hid_listen`](https://www.pjrc.com/teensy/hid_listen.html).
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md]:
```suggestion
Die Toolbox hört auf HID Nachrichten der usage pages `0xFF31` und `0X0074`. Dies ist kompatibel zu PJRC's [`hid_listen`](https://www.pjrc.com/teensy/hid_listen.html).
```

File: _i18n/de/toolbox.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+
+[Auf GitHub ansehen](https://github.com/qmk/qmk_toolbox)
+
+Dies ist eine Sammlung aus Flashing Werkzeugen, die in eine Applikation gepackt wurden. Es wird automatische Erkennung und automatisches Flashing von Firmware auf Tastaturen unterstützt.
+
+![](https://i.imgur.com/7bFh7vJ.png)
+
+## Flashing
+
+QMK Toolbox unterstützt die folgenden Bootloader:
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md]:
```suggestion
QMK Toolbox unterstützt folgende Bootloader:
```

File: _i18n/de/toolbox.md [hunk 18]
```diff
@@ -0,0 +1,59 @@
+---
+title: "QMK Toolbox"
+layout: qmk-editable
+permalink: /de/toolbox/
+---
+# QMK Toolbox
+
+[Auf GitHub ansehen](https://github.com/qmk/qmk_toolbox)
+
+Dies ist eine Sammlung aus Flashing Werkzeugen, die in eine Applikation gepackt wurden. Es wird automatische Erkennung und automatisches Flashing von Firmware auf Tastaturen unterstützt.
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md]:
```suggestion
Die QMK Toolbox vereint eine Sammlung von Flashing Werkzeugen in einer Applikation. Sie kann Tastaturen automatisch erkennen und ermöglicht das Flashes von Firmware auf Tastaturen.
```

File: _i18n/de/toolbox.md [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+ - AVRISP (Arduino ISP)
+ - USBasp (AVR ISP)
+
+Sollte Interesse für andere bestehen, können sie hinzugefügt werden, wenn ihre Befehle bekannt sind. 
+
+## HID Konsole
+
+Die Toolbox horcht auch auf HID Nachrichten auf usage page `0xFF31` und usage `0X0074`. Dies ist kompatibel mit PJRC's [`hid_listen`](https://www.pjrc.com/teensy/hid_listen.html).
+
+Sollten Sie `CONSOLE_ENABLE = yes` in der `rules.mk`-Datei ihrer Tastatur sein, können Sie Nachrichten mit `xprintf()` ausgeben. Dies ist zum Debuggen hilfreich:
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md]:
```suggestion
Sollte  `CONSOLE_ENABLE = yes` in der `rules.mk`-Datei ihrer Tastatur aktiv sein, können Sie Nachrichten mit `xprintf()` ausgeben. Dies ist zum Debuggen hilfreich:
```

File: _i18n/de/toolbox.md [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+
+![](https://i.imgur.com/03xuJhU.png)
+
+Sehen Sie in der [QMK Dokumentation](https://docs.qmk.fm/#/newbs_testing_debugging?id=debugging) für weitere Informationen.
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md]:
```suggestion
Wenden Sie sich an die [QMK Dokumentation](https://docs.qmk.fm/#/newbs_testing_debugging?id=debugging) für weitere Informationen.
```

File: _i18n/de/toolbox.md [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+
+## Installation
+
+### Abhängigkeiten
+
+Wenn Sie die QMK Toolbox auf Windows verwenden, werden Sie beim ersten Aufruf dazu aufgefordert die nötigen Treiber zu installieren. [Hier](https://github.com/qmk/qmk_driver_installer/releases) können Sie die letzte Version des QMK Treibers herunterladen.
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md]:
```suggestion
Wenn Sie die QMK Toolbox auf Windows verwenden, werden Sie beim ersten Aufruf gefragt, ob Sie die nötigen Treiber installieren möchten. [Hier](https://github.com/qmk/qmk_driver_installer/releases) können Sie die letzte Version des QMK Treibers herunterladen.
```

**Review by kuchosauronad0 — COMMENTED** [review]
Since we both accidentally translated the same pages independent of each other I tried to merge the best of both into this one since you were already using the formal pronouns (although I don't like it tbh, but I cant find my comment with the reasoning of it atm).

Just a tip: you can use the batch function for suggestions from the `Files changed`-tab, so that you don't have to wait for the page to reload each time :)

**Comment by noroadsleft** [comment]
@john2ksonn https://github.com/qmk/qmk.fm/pull/85#discussion_r339807145 ?

File: _i18n/de/toolbox.md [hunk 22]
```diff
@@ -0,0 +1,59 @@
+---
+title: "QMK Toolbox"
+layout: qmk-editable
+permalink: /de/toolbox/
+---
+# QMK Toolbox
+
+[Auf GitHub ansehen](https://github.com/qmk/qmk_toolbox)
+
+Die QMK Toolbox vereint eine Sammlung von Flashing Werkzeugen in einer Applikation. Sie kann Tastaturen automatisch erkennen und ermöglicht das Flashes von Firmware auf Tastaturen.
```
Comment (kuchosauronad0) [inline _i18n/de/toolbox.md:10]:
```suggestion
Die QMK Toolbox vereint eine Sammlung von Flashing Werkzeugen in einer Applikation. Sie kann Tastaturen automatisch erkennen und ermöglicht das Flashen von Firmware auf Tastaturen.
```

**Comment by noroadsleft** [comment]
Going to shove this one across the line.

**Comment by noroadsleft** [comment]
Thanks!

**Review by noroadsleft — APPROVED** [review]

**Pull request merged**